functional_testing/Grid_API_Testing/api_testing/testcases/basic_tests/test_nodeid_apis.py

```diff
@@ -107,60 +121,49 @@ def test005_get_job_details(self):
         #. Send get /nodes/{nodeid}/jobs/{jobid} api request.
         #. Compare response with the golden values.
         """
-
-        self.lg.info('Choose one random node of list of running nodes.')
-        node_id = self.base_test.get_random_node()
-
         self.lg.info('Get list of jobs of this node .')
-        response = self.nodes_api.get_nodes_nodeid_jobs(node_id=node_id)
+        response = self.nodes_api.get_nodes_nodeid_jobs(node_id=self.node_id)
         self.assertEqual(response.status_code, 200)
 
         self.lg.info('Choose one of these jobs to list its details.')
         jobs_list = response.json()
         job_id = jobs_list[random.randint(0, len(jobs_list))]['id']
```

Comment:
there is no jobs_list[len(job_list)] item. Please, set the randint limits to be from 0 to len(jobs_list)-1